puppet/modules/jenkins_job_builder/files/centos.org/jobs/foreman-pipelines.yml

```diff
@@ -47,6 +48,7 @@
       - '2.1'
     action:
       - install
+      - upgrade
```

Comment:
this will also generate el8 upgrade jobs on ci.centos.org, but nothing will ever trigger them :)